code/modules/cargo/markets/market_items/misc.dm

```diff
@@ -172,3 +172,33 @@
 	stock_min = 3
 	stock_max = 8
 	availability_prob = 90
+
+/datum/market_item/misc/poster/girlypop
```

Comment:
you don't need the /poster in the path, you're adding another type to the system that doesn't have any definition.

Reply:
I've adjusted this, thanks